src/main/java/com/gluonhq/substrate/model/ProjectConfiguration.java

```diff
@@ -27,51 +27,30 @@
  */
 package com.gluonhq.substrate.model;
 
-import com.gluonhq.substrate.Constants;
-
-import java.nio.file.Path;
-import java.nio.file.Paths;
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
 
 /**
- * This class contains all configuration info about the current project (not about the current OS/Arch/vendor etc)
- *
- * This class allows to specify where the Java core libs are located. If <code>setJavaStaticLibs</code> is called,
- * the libraries are expected to be found in the location provided by the location passed to
- * <code>setJavaStaticLibs</code>
- *
- * If this method has not been called, getJavaStaticLibsPath() will return the default location, taking into account
- * the value of javaStaticSdkVersion. If that value is not set, the default value is used.
+ * This class contains all public configuration info about the current project (not about the current OS/Arch/vendor etc)
+ * Instances of this class give hints to where real values should be contained from.
  */
 public class ProjectConfiguration {
```

Comment:
Ok, so now this will be the `PublicConfiguration`? Maybe we can rename it to `UserProjectConfiguration` or `CustomizableProjectConfiguration`?

Doing so will allow keep the name `ProjectConfiguration` for the `InternalProjectConfiguration`.